packages/fxa-settings/src/pages/SigninReported/en.ftl

```diff
@@ -0,0 +1,2 @@
+signin-reported-header = Thank you for your vigilance
```

Comment:
Could you add in a group comment for these strings to provide more context on when a user might encounter these strings/what kind of report the user has submitted? Things like `vigilance`, `Reports like this`, and `intruders` may end up being mistranslated or have incorrect nuance otherwise.

Reply:
I will -- thank you!